segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/hdfs/HDFSStorage.java

```diff
@@ -396,6 +396,10 @@ public SegmentHandle openWrite(String streamSegmentName) throws StreamSegmentExc
                         } catch (FileNotFoundException e) {
                             //This happens when more than one host is trying to fence and only one of the host goes through.
                             //Retry the rename so that host with the highest epoch gets access.
+                            //In the worst case, the current owner of the segment will win this race after retries equal
+                            //to the number of segmentstores in the race. Total number of segmentstores in the deployment
+                            // is the higher bound for this number.
+                            //It is safe to retry for MAX_INT times as we are sure that the loop will never go that long.
```

Comment:
We do not retry MAX_INT times - fix the comment.

Reply:
Fixed.